plugins/woocommerce-admin/client/core-profiler/index.tsx

```diff
@@ -100,8 +132,9 @@ export type CoreProfilerStateMachineContext = {
 	geolocatedLocation: {
 		location: string;
 	};
-	extensionsAvailable: ExtensionList[ 'plugins' ] | [];
-	extensionsSelected: string[]; // extension slugs
```

Comment:
Thanks for renaming `extensions` to `plugin`! I've also been confused about using both of these names.

Reply:
Thanks! I think it was just me being confused 😀

Reply:
Oh actually its called the extensions page in Figma

Reply:
@rjchow We also call it [extensions](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/data/src/onboarding/types.ts#L74) in onboarding package for some reasons 🤔